src/LinkedListVisualizer/LinkedListVisualizer.jsx

```diff
@@ -0,0 +1,497 @@
+
+        return(
+            <div className="heightInside">
+                <h1 align="center">
+                    Linked List Cycle Visualization
+                </h1>
+
+            <center>
+
+            <form onSubmit={this.handleAddNode} className="flexRowButton">
+                {inputGrid.map((inputBox, idx) => (
+                    <div>
+                        <input
+                            type="number"
+                            placeholder={`Insert Node Data`}
+                            value={inputBox.data}
+                            onChange={this.handleNodeDataChange(idx)}
+                            >
+                        </input>
+                    </div>
+                ))}
+
+                <button
+                    type="button"
+                    onClick={this.handleAddNode}
+                    >
+                    Add Node
+                </button>
+            </form>
+
+
+            <form onSubmit={this.handleAddCycle} className="flexRowButton">
+                    <div>
+                        <input
+                            type="number"
+                            placeholder={`Cycle Number Origin`}
+                            value={this.state.cycleToHolder}
+                            onChange={this.handleCycleBox}
+                            >
+                        </input>
+                    </div>
+
+
+                <button
+                    type="button"
+                    onClick={this.handleAddCycle}
+                    >
+                    Create Cycle
+                </button>
+            </form>
+
+            <br></br>
+
+            <button onClick={() => this.visualizeTransversal()}>
+                      Visualize
+            </button>
+
+            <br></br>
+            <br></br>
+
+
+            <button
+                    type="button"
+                    onClick={this.handleClearBoard}
+                    >
+                    Clear Board
+            </button>
+           
+            </center>
+
+            <div className="pushLeft">
+            <div className="flexRow">
+                {grid.map((node) => {
+                    const {location, data, nextNode} = node;
+                    
+                    if(location === (numberOfNodes - 1) && this.state.cyclePresent){
+                        return(
+                            <div>
+
+                            <div className="flexRow">
+                            <Node 
+                            location={location}
+                            data={data}
+                            key={location}
+                            nextNode={nextNode}
+                            />
+                            </div>
+                            
+                            <div style={{height: 100, backgroundColor: "#435058", width: this.state.downArrowWidth, position: "relative", left: (this.state.nodeWidth - this.state.downArrowWidth) / 2 }}>
```

Comment:
Use a down-arrow component or similar for this (or at least write a `renderDownArrow` function)